system_tests/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM google/python
+
+# Inject secrets
+# TODO: Encrypt/decrypt secrets
+ADD data/ data/
+ENV GOOGLE_APPLICATION_CREDENTIALS /home/vmagent/app/data/cloud-python-runtime-qa-credentials.json
+
+# Get the source.
+RUN git clone https://github.com/GoogleCloudPlatform/google-cloud-python.git
+WORKDIR google-cloud-python
+
+# Checkout the latest release.
+RUN git checkout $(git describe --tags --abbrev 0)
+
+# Install gcloud so we can do test setup
+RUN apt-get update && apt-get install curl
+RUN curl https://sdk.cloud.google.com | bash
```

Comment:
As long as your goal isn't to actually test the SDK it should be fine. I doubt these commands are going to change on you much in practice though.

One other benefit of fetching these tarballs is that the dl.google.com server which is used by the install script has no SLO and will cause flakes eventually. The versioned tarballs are published to GCS, which will have far fewer flakes.

The App Engine team just switched some internal tests over because of flakiness introduced by dl.google.com.